pkg/runner/runner.go

```diff
@@ -297,6 +298,18 @@ func CreateClient(cfg *Config, promMetrics *api.Metrics) (*pgclient.Client, erro
 		return nil, err
 	}
 
+	if cfg.InstallExtensions {
+		// Only check for background workers if TimessaleDB is installed.
+		if notOk, err := isBGWLessThanDBs(conn); err != nil {
+			return nil, fmt.Errorf("create-client: are background less than dbs: %w", err)
+		} else if notOk {
+			log.Warn("msg", "Maximum background workers are less than the number of databases available. "+
```

Comment:
Let's make the message:

Maximum background worker setting is too low for the number of databases in your system. Please increase your timescaledb.max_background_workers setting. See https://docs.timescale.com/latest/getting-started/configuring#workers for more information.